src/main/java/factionsplusplus/constants/GroupRole.java

```diff
@@ -0,0 +1,29 @@
+package factionsplusplus.constants;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public enum GroupRole {
+    Member(0x1),
+    Laborer(0x2 | GroupRole.Member.getLevel()),
+    Officer(0x4 | GroupRole.Laborer.getLevel() | GroupRole.Member.getLevel()),
+    Owner(0x8 | GroupRole.Officer.getLevel() | GroupRole.Laborer.getLevel() | GroupRole.Member.getLevel());
```

Comment:
You can condense these down since inheriting an officer already has the mask for laborer and member